### DISCUSSION TITLE
Firefly wallet App and Laptop DNS problem

### DISCUSSION THREAD

**Discussion by eyeforgold** [iotaledger/firefly#7711] [Help & Support]
Dear community,

Having not used my firefly wallet software (latest version) for a long time, I now get a message that ‘Could not find DNS resolution for server’ when trying to test send to another wallet. This is today 13 Nov 2023.

Is anyone familiar with this and is there a solution coming?

**Comment by begonaalvarezd** [comment]
When you mean latest version, you mean that you are on the latest version after the software upgrade v2.0.3 for Firefly IOTA? The network was recently upgraded so just making sure you are using a compatible version

**Reply by eyeforgold** [reply]
Thanks for responding
Yes, i have 2.0.3

**Reply by begonaalvarezd** [reply]
Can you paste the error log? You can find it from settings > help > error log > copy

**Comment by eyeforgold** [comment]
Hej Begoña,

The error log is empty

Ed

**Comment by eyeforgold** [comment]
Status says, network unreachable
Yes on 2.0.3

**Reply by begonaalvarezd** [reply]
It looks like your machine either is not allowing to access the nodes (can happen with VPNs, firewalls, etc...), that the node you had configured is unreachable. You can try disabling if you have firewalls or VPNs or re-import the profile and see if the new one has the same issue, we have to improve the node configuration from logged out profiles

**Comment by eyeforgold** [comment]
Thanks, that might be it. Will let you know shortly

**Comment by eyeforgold** [comment]
Just checked, No VPN running.
I re-imported already earlier to solve but didn’t help.
iOs does not work. I can do a fresh download on my Mac and recover the wallet.
After dinner over here 😃

Ed

**Reply by begonaalvarezd** [reply]
The mobile app is deprecated and unmaintained, you should be using the desktop version only, you can download the latest version from here https://firefly.iota.org/

**Comment by eyeforgold** [comment]
Ok clear,

I have overwritten a fresh 2.0.3. desktop version.
Recovered my wallet with seed.
It opened the procedure. New password and pin.
I tried to send a small amount. Failed.
It now goes into a loop of entering pin and then password, then pin again and then password. In other words I cannot enter app anymore.

Error message: No Mnemonic stored.

I will now clear all data from desktop and recover the wallet again.

I do have a copy of the wallet on my desktop which I will now first try to use.

Will be back when done

Ed

**Reply by begonaalvarezd** [reply]
In order to purge all the data, including wiping existing profiles, you can remove the following folders to start fresh

- Windows: `%APPDATA%\Roaming\Firefly\__storage__/`
- MacOS:` $HOME/Library/Application\ Support/Firefly/__storage__/`
- Linux: `~/.config/Firefly/__storage__/`

**Comment by eyeforgold** [comment]
Thanks for your patience and assistance.
I managed to fully reinstall and recover wallet succesfully.
The good news is that I can again send to another wallet.

Firstly the mobile apps have been removed and secondly I suspect some mysterious conflict between versions on my desktop.

It took exactly 5 minutes between send and receive, which I consider positive.
Looking forward to your further development announcements and thanks again.
We can close this ticket.

Regards

Ed
